buildlib/tools/zombie_killer.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash -eE
 
+logger() {
```

Comment:
can we use the system "logger" command that will write to /var/log/messages and add timestamps etc?
https://man7.org/linux/man-pages/man1/logger.1.html

Reply:
Thanks Yossi, I'm learning something new every day. 
Modified accordingly.

Reply:
I think the bash function `logger` cannot be the same as `/bin/logger`, as it will call itself.

```
$ cat a.sh
#!/bin/bash
logger() {
    logger -h
}
logger
$ time bash a.sh
<ctrl-c>
real    0m5.192s
```

Reply:
Definitely, was just about to change it myself, thanks!